apps/web-giddh/src/app/contact/account-statement/account-statement.component.html

```diff
@@ -0,0 +1,444 @@
+                                    {{ commonLocaleData?.app_amount }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="sort-header-right text-right"
+                                >
+                                    <span
+                                        >{{
+                                            element?.voucherAmount.type === "DEBIT"
+                                                ? element?.voucherAmount?.amount
+                                                : "-"
+                                        }}
+                                    </span>
+                                </td>
+                            </ng-container>
+                            <ng-container matColumnDef="Payments">
+                                <th
+                                    mat-header-cell
+                                    *matHeaderCellDef
+                                    class="bg-black text-white sort-header-right text-right font-16 fw-900"
+                                >
+                                    {{ commonLocaleData?.app_payment }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="sort-header-right text-right"
+                                >
+                                    {{ element?.voucherAmount.type != "DEBIT" ? element?.voucherAmount?.amount : "-" }}
+                                </td>
+                            </ng-container>
+                            <ng-container matColumnDef="Balance">
+                                <th
+                                    mat-header-cell
+                                    *matHeaderCellDef
+                                    class="bg-black text-white sort-header-right text-right font-16 fw-900"
+                                >
+                                    {{ commonLocaleData?.app_balance }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="sort-header-right text-right"
+                                >
+                                    {{
+                                        element.closingBalance?.amount >= 0
+                                            ? (element.closingBalance?.type === "CREDIT" ? "-" : "") +
```

Comment:
```suggestion
                                            ? (element.closingBalance?.type === "CREDIT" ? "-" : "") +
```

Use enum for this "CREDIT" "DEBIT"